timon/commands.py

```diff
@@ -127,6 +127,11 @@ def mk_parser():
     sub_prs.add_argument(
             '-d', '--loop-delay', default="auto",
             help="specifies loop delay")
+    sub_prs.add_argument(
+            '-pl', '--paranoia-loop', action="store_true",
+            help=(
+                "Run timon with a classic trio loop that cleanly"
+                " restarts after 1 day"))
```

Comment:
use fstring and get felay from settings or wherever it's coming from

Reply:
done